.circleci/config.yml

```diff
@@ -97,15 +45,10 @@ jobs:
       - run:
           command: |
             npm run build
-            DATE=$(date '+%Y-%m-%d')
-            export BUILD_NUMBER=${DATE}.${CIRCLE_BUILD_NUM}
-            export GIT_REF="$CIRCLE_SHA1"
```

Comment:
the format of the version number is slightly different here. the new value computed in the dps orb uses just the first 7 digits of the commit hash. does this matter @jonhindle ?

Reply:
👍 😬